clappr/clappr-ios
# Auto play

#### Goal
Add auto play feature 

#### How to test
* Run sample and check if video plays after load
* Run tests
